amivapi/settings.py

```diff
@@ -62,11 +62,13 @@
 REMOTE_MAILING_LIST_ADDRESS = None
 REMOTE_MAILING_LIST_KEYFILE = None
 REMOTE_MAILING_LIST_DIR = './'  # Use home directory on remote by default
+# Signups via email (@email_blueprint.route('/delete_signup/<token>') in email_links.py)
+# DEFINITIVE_DELETE = ''
 
 # SMTP server defaults
 API_MAIL = 'api@amiv.ethz.ch'
 API_MAIL_SUBJECT = "[AMIV] {subject}"
-SMTP_HOST = 'localhost'
+SMTP_HOST = 'localhost' # None in case you want to accept that no mails get sent (local testing)
```

Comment:
Thanks, I understand the comment now. But what exactly is the problem with leaving it on localhost for local testing? Is there some error?

Reply:
I removed it because I haven't been able to reproduce any error.